src/components/FormFields/FormSelectField.js

```diff
@@ -76,7 +75,13 @@ const FormSelectField = asField(({ fieldState, fieldApi, ...props }) => {
         return ''
       }
     }
-  };
+  }
+
+  const changeValue = (value, action) =>  {
+    setError('');
+    setValue(value);
+    onChange ? onChange(value) : setValue(value);
```

Comment:
Itt igazából ha nincs onChange a props-ban, akkor kétszer fogod meghívni a setValue-t. Igazából a lényeg pont az volt, hogy ha van onChange akkor meg kell hívnod azt is, ha meg nincs, akkor csak a setValue-t.